Manual DSR - split throws `not enough values to unpack` error

### Bug Description

```
2023-06-26 16:41:22 fides             | 2023-06-26 21:41:22.110 [DEBUG] (traversal:traverse:374): Found 1 end nodes: [__ROOT__:__ROOT__]
2023-06-26 16:41:22 fides             | 2023-06-26 21:41:22.119 [DEBUG] (task_resources:close:202): Closing all task resources for pri_b717a2a3-4cc1-42a6-b52a-86739038c691
2023-06-26 16:41:22 fides             | 2023-06-26 21:41:22.120 [INFO] (request_runner_service:upload_access_results:201): No results returned for access request pri_b717a2a3-4cc1-42a6-b52a-86739038c691
2023-06-26 16:41:22 fides             | 2023-06-26 21:41:22.122 [INFO] (application_config:get_resolved_config_property:180): No API-set storage.active_default_storage_type property found
2023-06-26 16:41:22 fides             | 2023-06-26 21:41:22.126 [INFO] (filter_results:filter_data_categories:32): Filtering Access Request results to return fields associated with data categories
2023-06-26 16:41:22 fides             | 2023-06-26 21:41:22.126 [INFO] (request_runner_service:upload_access_results:223): Starting access request upload for rule default_access_policy_rule for privacy request pri_b717a2a3-4cc1-42a6-b52a-86739038c691
2023-06-26 16:41:22 fides             | 2023-06-26 21:41:22.127 [INFO] (storage:write_to_in_memory_buffer:71): Writing data to in-memory buffer
2023-06-26 16:41:22 fides             | 2023-06-26 21:41:22.142 [ERROR] (logger:_log_exception:32): not enough values to unpack (expected 2, got 1)
2023-06-26 16:41:22 fides             | Traceback (most recent call last):
2023-06-26 16:41:22 fides             | 
2023-06-26 16:41:22 fides             |   File "/fides/src/fides/api/util/wrappers.py", line 12, in wrap
2023-06-26 16:41:22 fides             |     loop = asyncio.get_running_loop()
2023-06-26 16:41:22 fides             |            │       └ <built-in function get_running_loop>
2023-06-26 16:41:22 fides             |            └ <module 'asyncio' from '/usr/local/lib/python3.10/asyncio/__init__.py'>
2023-06-26 16:41:22 fides             | 
2023-06-26 16:41:22 fides             | RuntimeError: no running event loop
2023-06-26 16:41:22 fides             | 
2023-06-26 16:41:22 fides             | 
2023-06-26 16:41:22 fides             | During handling of the above exception, another exception occurred:
2023-06-26 16:41:22 fides             | 
2023-06-26 16:41:22 fides             | 
2023-06-26 16:41:22 fides             | Traceback (most recent call last):
2023-06-26 16:41:22 fides             | 
2023-06-26 16:41:22 fides             |   File "/usr/local/lib/python3.10/threading.py", line 973, in _bootstrap
2023-06-26 16:41:22 fides             |     self._bootstrap_inner()
2023-06-26 16:41:22 fides             |     │    └ <function Thread._bootstrap_inner at 0xffffa5cd0dc0>
2023-06-26 16:41:22 fides             |     └ <WorkerThread(AnyIO worker thread, started 281472335340000)>
2023-06-26 16:41:22 fides             |   File "/usr/local/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
2023-06-26 16:41:22 fides             |     self.run()
2023-06-26 16:41:22 fides             |     │    └ <function WorkerThread.run at 0xffff7d3dd2d0>
2023-06-26 16:41:22 fides             |     └ <WorkerThread(AnyIO worker thread, started 281472335340000)>
2023-06-26 16:41:22 fides             |   File "/opt/fides/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 807, in run
2023-06-26 16:41:22 fides             |     result = context.run(func, *args)
2023-06-26 16:41:22 fides             |              │       │   │      └ ()
2023-06-26 16:41:22 fides             |              │       │   └ functools.partial(<function resume_privacy_request_from_requires_input at 0xffff81beb0a0>, db=<sqlalchemy.orm.session.Extende...
2023-06-26 16:41:22 fides             |              │       └ <method 'run' of '_contextvars.Context' objects>
2023-06-26 16:41:22 fides             |              └ <_contextvars.Context object at 0xffff6124ed00>
2023-06-26 16:41:22 fides             | 
2023-06-26 16:41:22 fides             |   File "/fides/src/fides/api/api/v1/endpoints/privacy_request_endpoints.py", line 1573, in resume_privacy_request_from_requires_input
2023-06-26 16:41:22 fides             |     queue_privacy_request(
2023-06-26 16:41:22 fides             |     └ <function queue_privacy_request at 0xffff836d27a0>
2023-06-26 16:41:22 fides             | 
2023-06-26 16:41:22 fides             |   File "/fides/src/fides/api/service/privacy_request/request_runner_service.py", line 257, in queue_privacy_request
2023-06-26 16:41:22 fides             |     task = run_privacy_request.delay(
2023-06-26 16:41:22 fides             |            └ <@task: fides.api.service.privacy_request.request_runner_service.run_privacy_request of fides.api.tasks at 0xffffa0f732b0>
2023-06-26 16:41:22 fides             | 
2023-06-26 16:41:22 fides             |   File "/opt/fides/lib/python3.10/site-packages/celery/app/task.py", line 425, in delay
2023-06-26 16:41:22 fides             |     return self.apply_async(args, kwargs)
2023-06-26 16:41:22 fides             |            │    │           │     └ {'privacy_request_id': 'pri_b717a2a3-4cc1-42a6-b52a-86739038c691', 'from_webhook_id': None, 'from_step': None}
2023-06-26 16:41:22 fides             |            │    │           └ ()
2023-06-26 16:41:22 fides             |            │    └ <function Task.apply_async at 0xffffa0ffe440>
2023-06-26 16:41:22 fides             |            └ <@task: fides.api.service.privacy_request.request_runner_service.run_privacy_request of fides.api.tasks at 0xffffa0f732b0>
2023-06-26 16:41:22 fides             |   File "/opt/fides/lib/python3.10/site-packages/celery/app/task.py", line 572, in apply_async
2023-06-26 16:41:22 fides             |     return self.apply(args, kwargs, task_id=task_id or uuid(),
2023-06-26 16:41:22 fides             |            │    │     │     │               │          └ <function uuid at 0xffffa11d2dd0>
2023-06-26 16:41:22 fides             |            │    │     │     │               └ None
2023-06-26 16:41:22 fides             |            │    │     │     └ {'privacy_request_id': 'pri_b717a2a3-4cc1-42a6-b52a-86739038c691', 'from_webhook_id': None, 'from_step': None}
2023-06-26 16:41:22 fides             |            │    │     └ []
2023-06-26 16:41:22 fides             |            │    └ <function Task.apply at 0xffffa0ffe680>
2023-06-26 16:41:22 fides             |            └ <@task: fides.api.service.privacy_request.request_runner_service.run_privacy_request of fides.api.tasks at 0xffffa0f732b0>
2023-06-26 16:41:22 fides             |   File "/opt/fides/lib/python3.10/site-packages/celery/app/task.py", line 793, in apply
2023-06-26 16:41:22 fides             |     ret = tracer(task_id, args, kwargs, request)
2023-06-26 16:41:22 fides             |           │      │        │     │       └ {'id': 'a5cc883e-e235-42aa-81cf-2f959a463726', 'retries': 0, 'is_eager': True, 'logfile': None, 'loglevel': 0, 'hostname': '5...
2023-06-26 16:41:22 fides             |           │      │        │     └ {'privacy_request_id': 'pri_b717a2a3-4cc1-42a6-b52a-86739038c691', 'from_webhook_id': None, 'from_step': None}
2023-06-26 16:41:22 fides             |           │      │        └ ()
2023-06-26 16:41:22 fides             |           │      └ 'a5cc883e-e235-42aa-81cf-2f959a463726'
2023-06-26 16:41:22 fides             |           └ <function build_tracer.<locals>.trace_task at 0xffff62ace320>
2023-06-26 16:41:22 fides             |   File "/opt/fides/lib/python3.10/site-packages/celery/app/trace.py", line 451, in trace_task
2023-06-26 16:41:22 fides             |     R = retval = fun(*args, **kwargs)
2023-06-26 16:41:22 fides             |                  │    │       └ {'privacy_request_id': 'pri_b717a2a3-4cc1-42a6-b52a-86739038c691', 'from_webhook_id': None, 'from_step': None}
2023-06-26 16:41:22 fides             |                  │    └ ()
2023-06-26 16:41:22 fides             |                  └ <bound method run_privacy_request of <@task: fides.api.service.privacy_request.request_runner_service.run_privacy_request of ...
2023-06-26 16:41:22 fides             | 
2023-06-26 16:41:22 fides             |   File "/fides/src/fides/api/util/wrappers.py", line 15, in wrap
2023-06-26 16:41:22 fides             |     return asyncio.run(func(*args, **kwargs))
2023-06-26 16:41:22 fides             |            │       │   │     │       └ {'privacy_request_id': 'pri_b717a2a3-4cc1-42a6-b52a-86739038c691', 'from_webhook_id': None, 'from_step': None}
2023-06-26 16:41:22 fides             |            │       │   │     └ (<@task: fides.api.service.privacy_request.request_runner_service.run_privacy_request of fides.api.tasks at 0xffffa0f732b0>,)
2023-06-26 16:41:22 fides             |            │       │   └ <function run_privacy_request at 0xffff836d2830>
2023-06-26 16:41:22 fides             |            │       └ <function run at 0xffffa5ae5ea0>
2023-06-26 16:41:22 fides             |            └ <module 'asyncio' from '/usr/local/lib/python3.10/asyncio/__init__.py'>
2023-06-26 16:41:22 fides             | 
2023-06-26 16:41:22 fides             |   File "/usr/local/lib/python3.10/asyncio/runners.py", line 44, in run
2023-06-26 16:41:22 fides             |     return loop.run_until_complete(main)
2023-06-26 16:41:22 fides             |            │    │                  └ <coroutine object run_privacy_request at 0xffff63904970>
2023-06-26 16:41:22 fides             |            │    └ <method 'run_until_complete' of 'uvloop.loop.Loop' objects>
2023-06-26 16:41:22 fides             |            └ <uvloop.Loop running=True closed=False debug=False>
2023-06-26 16:41:22 fides             | 
2023-06-26 16:41:22 fides             | > File "/fides/src/fides/api/service/privacy_request/request_runner_service.py", line 366, in run_privacy_request
2023-06-26 16:41:22 fides             |     access_result_urls = upload_access_results(
2023-06-26 16:41:22 fides             |                          └ <function upload_access_results at 0xffff836a8ca0>
2023-06-26 16:41:22 fides             | 
2023-06-26 16:41:22 fides             |   File "/fides/src/fides/api/service/privacy_request/request_runner_service.py", line 229, in upload_access_results
2023-06-26 16:41:22 fides             |     download_url: Optional[str] = upload(
2023-06-26 16:41:22 fides             |                   │               └ <function upload at 0xffff83db3b50>
2023-06-26 16:41:22 fides             |                   └ typing.Optional
2023-06-26 16:41:22 fides             | 
2023-06-26 16:41:22 fides             |   File "/fides/src/fides/api/service/storage/storage_uploader_service.py", line 38, in upload
2023-06-26 16:41:22 fides             |     return uploader(db, config, data, privacy_request)
2023-06-26 16:41:22 fides             |            │        │   │       │     └ <fides.api.models.privacy_request.PrivacyRequest object at 0xffff637dfdf0>
2023-06-26 16:41:22 fides             |            │        │   │       └ defaultdict(<class 'list'>, {'manual': [{'a': 'b'}]})
2023-06-26 16:41:22 fides             |            │        │   └ <fides.api.models.storage.StorageConfig object at 0xffff635b2c50>
2023-06-26 16:41:22 fides             |            │        └ <sqlalchemy.orm.session.ExtendedSession object at 0xffff636360e0>
2023-06-26 16:41:22 fides             |            └ <function _local_uploader at 0xffff83a508b0>
2023-06-26 16:41:22 fides             | 
2023-06-26 16:41:22 fides             |   File "/fides/src/fides/api/service/storage/storage_uploader_service.py", line 97, in _local_uploader
2023-06-26 16:41:22 fides             |     return upload_to_local(data, file_key, privacy_request, config.format.value)  # type: ignore
2023-06-26 16:41:22 fides             |            │               │     │         │                │      └ <sqlalchemy.orm.attributes.InstrumentedAttribute object at 0xffffa0f33c40>
2023-06-26 16:41:22 fides             |            │               │     │         │                └ <fides.api.models.storage.StorageConfig object at 0xffff635b2c50>
2023-06-26 16:41:22 fides             |            │               │     │         └ <fides.api.models.privacy_request.PrivacyRequest object at 0xffff637dfdf0>
2023-06-26 16:41:22 fides             |            │               │     └ 'pri_b717a2a3-4cc1-42a6-b52a-86739038c691.zip'
2023-06-26 16:41:22 fides             |            │               └ defaultdict(<class 'list'>, {'manual': [{'a': 'b'}]})
2023-06-26 16:41:22 fides             |            └ <function upload_to_local at 0xffff83a505e0>
2023-06-26 16:41:22 fides             | 
2023-06-26 16:41:22 fides             |   File "/fides/src/fides/api/tasks/storage.py", line 180, in upload_to_local
2023-06-26 16:41:22 fides             |     in_memory_file = write_to_in_memory_buffer(resp_format, data, privacy_request)
2023-06-26 16:41:22 fides             |                      │                         │            │     └ <fides.api.models.privacy_request.PrivacyRequest object at 0xffff637dfdf0>
2023-06-26 16:41:22 fides             |                      │                         │            └ defaultdict(<class 'list'>, {'manual': [{'a': 'b'}]})
2023-06-26 16:41:22 fides             |                      │                         └ 'html'
2023-06-26 16:41:22 fides             |                      └ <function write_to_in_memory_buffer at 0xffff83a50310>
2023-06-26 16:41:22 fides             | 
2023-06-26 16:41:22 fides             |   File "/fides/src/fides/api/tasks/storage.py", line 103, in write_to_in_memory_buffer
2023-06-26 16:41:22 fides             |     ).generate()
2023-06-26 16:41:22 fides             | 
2023-06-26 16:41:22 fides             |   File "/fides/src/fides/api/service/privacy_request/dsr_package/dsr_report_builder.py", line 158, in generate
2023-06-26 16:41:22 fides             |     [dataset_name, collection_name] = key.split(":")
2023-06-26 16:41:22 fides             |                                       │   └ <method 'split' of 'str' objects>
2023-06-26 16:41:22 fides             |                                       └ 'manual'
2023-06-26 16:41:22 fides             | 
2023-06-26 16:41:22 fides             | ValueError: not enough values to unpack (expected 2, got 1)
2023-06-26 16:41:22 fides             | 2023-06-26 21:41:22.164 [INFO] (main:log_request:140): Request received | {'method': 'POST', 'status_code': 200, 'handler_time': '129.195ms', 'path': '/api/v1/privacy-request/pri_b717a2a3-4cc1-42a6-b52a-86739038c691/resume_from_requires_input'}
```

#### Steps to Reproduce

1. Set local storage destination to use `html` as the format
2. Create a manual integration and add in a field, I called it `a`
3. Create a privacy request
4. Enter in the manual data, I put in `b`
5. Submit DSR
6. Error that can't be reprocessed


### Expected behavior

Manual DSR shouldn't have errored it out

### Screenshots

_If applicable, add screenshots to help explain your problem._

### Environment

* Version:
* OS:
* Python Version:
* Docker Version:

### Additional context

_Add any other context about the problem here._

--- a/src/fides/api/service/privacy_request/dsr_package/dsr_report_builder.py
+++ b/src/fides/api/service/privacy_request/dsr_package/dsr_report_builder.py
@@ -155,7 +155,10 @@ class DsrReportBuilder:
             # pre-process data to split the dataset:collection keys
             datasets: Dict[str, Any] = defaultdict(lambda: defaultdict(list))
             for key, rows in self.dsr_data.items():
-                [dataset_name, collection_name] = key.split(":")
+                parts = key.split(":", 1)
+                dataset_name, collection_name = (
+                    parts if len(parts) > 1 else ("manual", parts[0])
+                )
                 datasets[dataset_name][collection_name].extend(rows)
 
             for dataset_name, collections in datasets.items():

--- a/tests/ops/service/test_storage_uploader_service.py
+++ b/tests/ops/service/test_storage_uploader_service.py
@@ -354,6 +354,7 @@ class TestWriteToInMemoryBuffer:
                 {"uuid": "xyz-122-333", "name": "foo1", "email": "foo@bar1"},
             ],
             "mongo:foobar": [{"_id": 1, "customer": {"x": 1, "y": [1, 2]}}],
+            "filing_cabinet": [{"id": "123"}],  # represents manual data
         }
 
     def test_json_data(self, data, privacy_request):
@@ -367,6 +368,7 @@ class TestWriteToInMemoryBuffer:
 
         zipfile = ZipFile(buff)
         assert zipfile.namelist() == [
+            "filing_cabinet.csv",
             "mongo:address.csv",
             "mysql:customer.csv",
             "mongo:foobar.csv",
@@ -435,6 +437,9 @@ class TestWriteToInMemoryBuffer:
         assert zipfile.namelist() == [
             "/main.css",
             "/back.svg",
+            "/manual/index.html",
+            "/manual/filing_cabinet/1.html",
+            "/manual/filing_cabinet/index.html",
             "/mongo/address/1.html",
             "/mongo/address/2.html",
             "/mongo/address/index.html",

--- a/tests/ops/service/test_storage_uploader_service.py
+++ b/tests/ops/service/test_storage_uploader_service.py
@@ -368,10 +368,10 @@ class TestWriteToInMemoryBuffer:
 
         zipfile = ZipFile(buff)
         assert zipfile.namelist() == [
-            "filing_cabinet.csv",
             "mongo:address.csv",
             "mysql:customer.csv",
             "mongo:foobar.csv",
+            "filing_cabinet.csv",
         ]
 
         with zipfile.open("mongo:address.csv") as address_csv:
@@ -437,9 +437,6 @@ class TestWriteToInMemoryBuffer:
         assert zipfile.namelist() == [
             "/main.css",
             "/back.svg",
-            "/manual/index.html",
-            "/manual/filing_cabinet/1.html",
-            "/manual/filing_cabinet/index.html",
             "/mongo/address/1.html",
             "/mongo/address/2.html",
             "/mongo/address/index.html",
@@ -450,6 +447,9 @@ class TestWriteToInMemoryBuffer:
             "/mysql/customer/2.html",
             "/mysql/customer/index.html",
             "/mysql/index.html",
+            "/manual/filing_cabinet/1.html",
+            "/manual/filing_cabinet/index.html",
+            "/manual/index.html",
             "/index.html",
         ]
 

